File: RUBY/Cap07/c07ex05.rb
# c07ex05.rb

#!/usr/bin/env ruby

def fatorial(n, fat)
  for i in 1..n
    fat[0] *= i
  end
end

resp = [1]

print "Qual fatorial: "
limite = gets.chomp.to_i

fatorial(limite, resp)

puts "Fatorial = #{resp[0]}"

print "\nTecle <Enter> para encerrar... "
gets

File: PYTHON/Cap07/c07ex05.py
# c07ex05.py

def fatorial(n, fat):
  for i in range(1, n+1):
    fat[0] = fat[0] * i

resp = [1]

print("Qual fatorial: ", end="")
limite = int(input())

fatorial(limite, resp)

print(f"Fatorial = {resp[0]}")

input("\nTecle <Enter> para encerrar... ")
